pkg/vex/vulnerability.go

```diff
@@ -0,0 +1,20 @@
+/*
+Copyright 2023 The OpenVEX Authors
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package vex
+
+// Vulnerability is a struct that captures the vulnerability identifier and
+// its aliases. When defined, the ID field should be an IRI.
+type Vulnerability struct {
+	ID          string            `json:"@id,omitempty"`
+	Name        VulnerabilityID   `json:"name,omitempty"`
+	Description string            `json:"description,omitempty"`
+	Aliases     []VulnerabilityID `json:"aliases,omitempty"`
+}
+
+// VulnerabilityID is a string that captures a vulnerability identifier. It is
+// a free form string but it is intended to capture the identifiers used by
+// tracking systems.
+type VulnerabilityID string
```

Comment:
What is the thinking for having this be its own type instead of just using `string` as is?

Reply:
Mainly overloading it with methods later, validating, them, etc.